sources/zend/Kaltura/Client/ParseUtils.php

```diff
@@ -134,6 +134,15 @@ public static function jsObjectToClientObject($value, $fallbackType = null)
 				{
 					continue;
 				}
+
+				if($attribute === 'relatedObjects') {
+					$object->relatedObjects = [];
```

Comment:
I am not entirely sure, and I hope it's not the case anymore, but *if* we need to support pre-7 PHP, then you might need to change it to `array()`.